server.yml

```diff
@@ -1,12 +1,19 @@
 ---
+- hosts: localhost
+  tasks:
+    - name: Ensure environment is defined
+      fail:
+        msg: "Environment missing. Use `-e` to define `env`:\nansible-playbook server.yml -e env=<environment>"
+      when: env is not defined
+
```

Comment:
Great idea.

I added a commit to include the new `variable-check.yml` playbook. It's DRY but also a bit more complex to look at. The messages needed to differ slightly per playbook.

You'll see that I went with yaml:
```
- include: variable-check.yml
  vars:
    playbook: server.yml
```
even though it could have been more succinct:
```
- include: variable-check.yml playbook=server.yml
```